training/upgrade-a-cluster.md

```diff
@@ -0,0 +1,142 @@
+---
+title: Upgrade a Cluster
```

Comment:
The section should mention the `SET CLUSTER VERSION` command.

Reply:
It looks like that isn't covered [`SET CLUSTER SETTING`](https://www.cockroachlabs.com/docs/stable/set-cluster-setting.html)––do you know who I can chat with about it to document it?

Reply:
Ah, that was an oversight on my part, @sploiselle. It'd documented in the [rolling upgrade instructions](https://www.cockroachlabs.com/docs/stable/upgrade-cockroach-version.html#finalize-the-upgrade), but not in `SET CLUSTER SETTING`. Want me to take care of this?